modules/bbp-morphology/src/main/resources/schemas/bbp/morphology/reconstructedcell/v0.1.0.json

```diff
@@ -0,0 +1,158 @@
+              "name" : "Distance to boundary",
+              "description" : "Distance of the reconstructed cell to pia or to another brain boundary.",
+              "node" : "bbplocsh:DistanceToBoundaryShape",
+              "maxCount" : 1
+            },
+            {
+              "path" : "nsg:mType",
+              "name" : "Morphology type",
+              "description" : "The morphology type of the reconstructed cell: need the cell type for class restriction.",
+              "node": "bbptlotsh:CellTypeOntologyTermShape",
+              "maxCount" : 1
+            },
+            {
+              "path" : "nsg:mSubType",
+              "name" : "Morphology sub-type",
+              "description" : "The morphology sub-type of the reconstructed cell: need the cell sub-type for class restriction.",
+              "datatype": "xsd:string",
+              "maxCount" : 1
+            },
+            {
+              "path" : "nsg:subRegion",
+              "name" : "Sub region",
+              "description" : "Sub region where the reconstructed cell is located, CA1a in hippocampus for example.",
+              "node":"bbptlotsh:BrainRegionOntologyTermShape",
+              "maxCount" : 1
+            },
+            {
+              "path" : "nsg:positionInLayer",
+              "name" : "Position in layer",
+              "description" : "Position in layer where the reconstructed cell is located, can be 'deep' or 'superficial' for hippocampus.",
+              "node":"bbplocsh:BrainLocationShape",
+              "maxCount" : 1
+            },
+            {
+              "path" : "nsg:longitudinalAxis",
+              "name" : "Longitudinal axis",
+              "description" : "Position in longitudinal axis where the reconstructed cell is located, can be 'dorsal' or 'ventral' for hippocampus.",
+              "node":"bbplocsh:BrainLocationShape",
+              "maxCount" : 1
+            },
+            {
+              "path" : "nsg:layer",
+              "name" : "Layer",
+              "description" : "Layer where the reconstructed cell is located.",
+              "node":"bbplocsh:BrainLocationShape",
+              "maxCount" : 1
+            },
+            {
+              "path" : "nsg:integrity",
+              "name" : "Integrity",
+              "description" : "Gives information on the integrity of the reconstructed cell.",
+              "node" : "nsg:IntegrityShape",
+              "maxCount" : 1
+            },
+            {
+              "path" : "nsg:somaType",
+              "name" : "Soma type",
+              "description" : "Gives information on the type of some of the reconstructed cell, values can be 3D or 2D contour, 1 point, 3 point.",
+              "sh:in" : {
+                "@list" : [ "3D contour", "2D contour", "1 point", "3 point" ]
+              },
+              "maxCount" : 1
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "@id" : "this:IntegrityShape",
```

Comment:
I would suggest to name this shape ReconstructedCellIntegerityShape.

Reply:
OK